packages/main/src/ToolbarSelect.ts

```diff
@@ -14,7 +14,7 @@ import ToolbarPopoverSelectTemplate from "./generated/templates/ToolbarPopoverSe
 import ToolbarItem from "./ToolbarItem.js";
 import Select from "./Select.js";
 import Option from "./Option.js";
-import "./ToolbarSelectOption.js";
+import ToolbarSelectOption from "./ToolbarSelectOption.js";
```

Comment:
Since ToolbarSelectOption is only used for type it can be imported as type

```ts
import type ToolbarSelectOption from "./ToolbarSelectOption.js";
```